proposals/lip-0022.md

```diff
@@ -177,25 +177,19 @@ Internally, the algorithm works as follows:
 
 In the case that `ListStandbyDelegates` contains less than 2 elements, the delegates with rank 102 and 103 will be chosen as `delegate1 `and `delegate2` respectively.
 
-### Round Computation and Delegate List Ordering
+### Round Computation and Delegate List
 
 The round computation is performed in the same way as in the current implementation with the difference that `delegate1` and `delegate2`, as defined in the previous section, are included in the forging delegate list of `round` + 1. This implies the following:
 
-*   The round length is now 103 block slots. Currently, this is defined by the constant  <code>[ACTIVE_DELEGATES](https://github.com/LiskHQ/lisk-sdk/blob/310283408102ff9f891830225afe8124e833a608/config/default/constants.js#L55)</code>.  For clarity, this constant should be replaced for three constants defining the number of the top 101 delegates by `delegateWeight`, the number of standby delegates, i.e., 2, and the length of the round (sum of the two previous constants).
+*   The round length is now 103 block slots. Currently, this is defined by the constant  <code>[ACTIVE_DELEGATES](https://github.com/LiskHQ/lisk-sdk/blob/310283408102ff9f891830225afe8124e833a608/config/default/constants.js#L55)</code>.  For clarity, this constant can be replaced for three constants defining the number of the top 101 delegates by `delegateWeight`, the number of standby delegates, i.e., 2, and the length of the round (sum of the two previous constants).
```

Comment:
I find this sentence pretty confusing, maybe something like:
"For clarity, this constant can be replaced by three constants: the number of top delegates by `delegateWeight`, i.e.,101, the number of standby delegates, i.e., 2, and the length of the round (sum of the two previous constants)."

Reply:
changed as proposed

Reply:
I think this bullet point should be removed. The constants are part of the protocol and these implementation details should rather be decided at time of implementation.

Reply:
Note that the statement "The round length is now 103 block slots." is incorrect as block slots can also be missed.

Reply:
good point about the round length. The actual protocol change is that a round has 103 blocks now. In the code currently this is specified by using the `ACTIVE_DELEGATES` but this has to change. I just tried to make that clear but I agree these details should be removed from the LIP.